lib/types/documentdb.d.ts

```diff
@@ -0,0 +1,10 @@
+import * as documentdb from "documentdb";
+
+declare module "documentdb" {
+  // [#153802058] _ts is a number but is defined as string.
+  // TODO: get rid of this file when this pull request gets merged:
```

Comment:
pls also open a story on pivotal

Reply:
we can leave the one referred in the comment above open:
https://www.pivotaltracker.com/story/show/153802058

Reply:
I would prefer to close that one (bug is fixed indeed) and add a different one as reminder... we don't know if/when they will merge it and I'd like to avoid keeping the bug story open...

Reply:
ok